SU2_CFD/src/output/CFlowCompOutput.cpp

```diff
@@ -254,8 +254,8 @@ void CFlowCompOutput::SetVolumeOutputFields(CConfig *config){
 
   if (config->GetViscous()) {
     AddVolumeOutput("LAMINAR_VISCOSITY", "Laminar_Viscosity", "PRIMITIVE", "Laminar viscosity");
-    AddVolumeOutput("THERMAL_CONDUCTIVITY", "Thermal_Conductivity", "PRIMITIVE", "Thermal conductivity");
-    AddVolumeOutput("HEAT_CAPACITY", "Heat_Capacity", "PRIMITIVE", "Heat Capacity");
+    AddVolumeOutput("THERMAL_CONDUCTIVITY", "Thermal_Conductivity", "FLUID_PROPERTIES", "Thermal conductivity");
+    AddVolumeOutput("HEAT_CAPACITY", "Heat_Capacity", "FLUID_PROPERTIES", "Heat Capacity");
```

Comment:
Add these after all the primitives were added. For solution that is a requirement otherwise we have issues with restarts. For primitives I don't think so but it would still be good to keep the convention.